integrationtests/as-integration-embedded/src/test/java/org/infinispan/test/integration/as/cdi/GreetingCacheManagerIT.java

```diff
@@ -37,7 +36,12 @@
 
    private static Asset manifest() {
       String manifest = Descriptors.create(ManifestDescriptor.class)
-            .attribute("Dependencies", "org.infinispan.cdi:" + Version.getModuleSlot() + " services, org.infinispan.jcache:" + Version.getModuleSlot() +" services").exportAsString();
+              .attribute(
+                      "Dependencies", "org.infinispan.cdi:" + Version.getModuleSlot() + " services, " +
+                              "org.infinispan.jcache:" + Version.getModuleSlot() + " services, " +
+                              //TODO: ISPN-5148 - remove this dependency as soon as WFLY-4250 is fixed.
+                              "org.infinispan:" + Version.getModuleSlot())
```

Comment:
This is potentially risky workaround. Should we go for Wildfly 8.2 or sort out WFLY-4250 first?